### ISSUE TITLE
Supply info values to fields created using buildSchema.

### ISSUE THREAD

**Issue by nripendra** [graphql/graphql-js#598]
Currently only args and context are provided:

```js
return source[fieldName](args, context); //defaultFieldResolver
```
It doesn't seem to be sufficient as we loose information like which fields are selected. Querying all the fields from data source sometimes can be expensive (e.g. binary fields).

**Comment by robzhu** [comment]
Can you please provide some more details around your use case?

**Comment by nripendra** [comment]
@robzhu 

I have registered handler using `buildSchema`, but don't get the list of fields that is selected.

**Comment by leebyron** [comment]
Closed by #615
